### ISSUE TITLE
NetMQContext's MaxSockets is wrong.

### ISSUE THREAD

**Issue by zhengchun** [zeromq/netmq#323]
hi,i found a MaxSockets property of NetMQContext class is not correct,
```c#
public int MaxSockets
{
    get { m_ctx.CheckDisposed(); return m_ctx.MaxSockets; }
    set { m_ctx.CheckDisposed(); m_ctx.IOThreadCount = value; }
}
```
should be change a `m_ctx.IOThreadCount = value` to `m_ctx.MaxSockets = value`,right?

**Comment by somdoron** [comment]
Yes. You are right, do you want to provide a pull request?

**Comment by drewnoakes** [comment]
Closed by #328.
